gp-cli/src/main/java/com/ibm/g11n/pipeline/tools/cli/CreateBundleCmd.java

```diff
@@ -31,7 +31,7 @@
 @Parameters(commandDescription = "Creates a new translation bundle.")
 final class CreateBundleCmd extends BundleCmd {
     @Parameter(
-            names = {"-l", "--language"},
+            names = {"-l", "--languages"},
```

Comment:
OK. I'd recommend `-l`/`--language` for both create and update (for consistency)